solutions/kcc-namespaces/README.md

```diff
@@ -4,7 +4,7 @@
 This solution is a simple fork of the KCC Project Namespaces blueprint found at
 https://cloud.google.com/anthos-config-management/docs/tutorials/project-namespace-blueprint
 
-This simple solution will great a dedicated namespace to manage your GCP resources in a certain project. The namespace and project-id will be match for simplicity. This solution assumes that you storing all high-level resources in the `config-control` namespsace.
+This simple solution will create a dedicated namespace to manage your GCP resources in a certain project. The namespace and project-id will be match for simplicity. This solution assumes that you storing all high-level resources in the `config-control` namespsace.
```

Comment:
might as well fix the word namespace at the end of this line

Reply:
Done